bridges/ExplosmBridge.php

```diff
@@ -1,22 +1,58 @@
 <?php
-class ExplosmBridge extends FeedExpander {
+class ExplosmBridge extends BridgeAbstract {
 
 	const MAINTAINER = 'bockiii';
 	const NAME = 'Explosm Bridge';
 	const URI = 'https://www.explosm.net/';
 	const CACHE_TIMEOUT = 4800; //2hours
 	const DESCRIPTION = 'Returns the last 5 comics';
+	const PARAMETERS = array(
+		'Get latest posts' => array(
+			'limit' => array(
+				'name' => 'Posts limit',
+				'type' => 'number',
+				'required' => true,
+				'title' => 'Maximum number of items to return',
+				'defaultValue' => 5
```

Comment:
Do we need "required" here?
This setting will break previous usage, where "/?action=display&bridge=Explosm&format=Html" worked. If I apply this change, users need to reedit their rss feeds.

Reply:
I did not think about it, this is a great insight. I makred it required becouse a limit is required. I could make it so the default is 5 which is the number of posts the bridge returned when it used feedburner before this change. Does the `'defaultValue' => 5` will take care of that, im not sure.

Reply:
Tested myself, i fixed this and tested for old feed urls to work.